flink-architecture-tests/flink-architecture-tests-production/pom.xml

```diff
@@ -39,35 +39,41 @@ under the License.
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-architecture-tests-base</artifactId>
+			<scope>compile</scope>
```

Comment:
Isn't `compile` already the default scope?

Reply:
no, default is `test`